app/models/activity.rb

```diff
@@ -3,6 +3,6 @@ class Activity < ApplicationRecord
   has_many :activity_teaching_methods, dependent: :destroy
   has_many :teaching_methods, through: :activity_teaching_methods
 
-  validates :lesson_part_id, presence: true
+  validates :lesson_part, presence: true
```

Comment:
I'm not sure this is necessary just yet, but eventually in the scope of `:publish`